src/hotspot/share/opto/compile.hpp

```diff
@@ -937,6 +937,7 @@ class Compile : public Phase {
   PhaseGVN*         initial_gvn()               { return _initial_gvn; }
   Unique_Node_List* for_igvn()                  { return _for_igvn; }
   inline void       record_for_igvn(Node* n);   // Body is after class Unique_Node_List.
+  inline void       remove_for_igvn(Node* n);   // Body is after class Unique_Node_List.
```

Comment:
May be change comment.

Reply:
Updated to say its in node.hpp. Though it is technically still after Unique_Node_List.